### DISCUSSION TITLE
Theme overrides: Which files are minimum

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1363] [Help] [answered]
Hi, assuming I want to change the config.json using a Theme overrides. Which files are needed in the overrides folder as a minimum?

**Comment by bobmitro** [comment]
Only the one you want to override.

**Reply by 3muchfun** [reply]
Ok, but if I put config.json in the override folder, only, I get the error "index.hbs not found" or similar, when I try to preview the site. 
You say, I made something wrong?

**Reply by bobmitro** [reply]
Overrides works very well. So I suspect there must be something wrong on your side. 
Just follow this instruction: https://getpublii.com/docs/overriding-publii-theme.html#howtouseoverridefiles I'm sure everything will work as it should.

**Reply by 3muchfun** [reply]
![Bildschirmfoto 2024-02-14 um 15 57 01](https://github.com/GetPublii/Publii/assets/17292904/f4b6a794-3f24-4a25-9627-6b23a32fc7d5)
that's what I did

**Reply by bobmitro** [reply, marked as answer]
It looks ok. Please email me the site backup; I will check it out.

**Reply by 3muchfun** [reply]
Well, I tried it again today, and it works like a charm. No idea, what (I) changed ...
